Amazon.Ion.ObjectMapper/IonObjectSerializer.cs

```diff
@@ -134,6 +135,11 @@ private PropertyInfo FindProperty(string readName)
                 return exact;
             }
 
+            if (options.PropertyNameCaseInsensitive)
+            {
+                return targetType.GetProperties().FirstOrDefault(p => String.Equals(p.Name, readName, StringComparison.CurrentCultureIgnoreCase));
```

Comment:
`OrdinalIgnoreCase` seems like the better option for StringComparison here.

Reply:
Done.